apps/web/__tests__/support/types/types.ts

```diff
@@ -5,7 +5,9 @@ export type Address = {
   state: string;
   country: string;
   zipcode: string;
+  zipCode: string;
```

Comment:
Do we need both of them?

Reply:
we need the added one, not sure about the other, let me check to see what I can find 
I'll remove it, if thats an option